Sources/RaftNIO/Log/FilePath+Raft.swift

```diff
@@ -53,12 +51,12 @@ extension FilePath {
     }
 
     /// Temporary methods to hide direct usage of FileManager
-    func isPathExist() -> Bool {
+    public func isPathExist() -> Bool {
         FileManager.default.fileExists(atPath: string)
     }
 
     @discardableResult
-    func createDirectory() throws -> Bool {
+    public func createDirectory() throws -> Bool {
         try FileManager.default.createDirectory(atPath: string, withIntermediateDirectories: true)
         return true
```

Comment:
should this be done at init() and the init be throwing?

Reply:
You mean at `FilePath` init?